back/app/services/permissions/phase_permissions_service.rb

```diff
@@ -61,7 +61,7 @@ def denied_reason_for_action(action, user, phase, project: phase&.project, react
       when 'taking_poll'
         taking_poll_denied_reason_for_phase(phase, user)
       else
-        raise "Unsupported action: #{action}"
+        raise "Unsupported action: #{action}" unless SUPPORTED_ACTIONS.include?(action)
```

Comment:
Because there are no denied reasons for 'attending_event' relating to the phase itself - the denied reasons all come from the user level - so added this so that we only get unsupported action when it's truly unsupported.